xformers/sparse/csr_tensor.py

```diff
@@ -183,6 +183,8 @@ def _masked_matmul(cls, a, b, mask):
         row_indices = mask.__row_indices
         row_offsets = mask.__row_offsets
         column_indices = mask.__column_indices
+        a = a.contiguous()
+        b = b.contiguous()
```

Comment:
This is not equivalent to what you had before actually, you want to do `b.transpose().contiguous()`